Source/WebCore/bindings/js/JSDOMConvertStrings.cpp

```diff
@@ -118,4 +147,104 @@ AtomString valueToUSVAtomString(JSGlobalObject& lexicalGlobalObject, JSValue val
     return replaceUnpairedSurrogatesWithReplacementCharacter(WTFMove(string));
 }
 
+// https://w3c.github.io/trusted-types/dist/spec/#process-value-with-a-default-policy-algorithm
+static inline std::variant<std::nullptr_t, Ref<TrustedHTML>, Ref<TrustedScript>, Ref<TrustedScriptURL>> processValueWithDefaultPolicy(VM& vm, ScriptExecutionContext& scriptExecutionContext, const String& expectedType, const String& input, const String& sink)
```

Comment:
This is a static inline rather than being declared in the header because I can't import the Trusted types headers into the header file and std::variant can't make use of forward declarations. In future this function will probably end up in a separate helper file as it will be used in more places.